src/components/productCard/productCard.js

```diff
@@ -13,7 +37,7 @@ export default function ProductCard({ product }) {
                     <b>Rs. {price}</b>
 
                 </div>
-                <button className="btn login cart">Add to cart</button>
+                <button onClick={handleAddToCart} className={added ? "btn move-btn cart" : "btn login cart"}>{added ? "Added" : "Add to cart"}</button>
```

Comment:
SUGGESTION: If the product is added to the cart, we can show to go to cart button and navigate the user to cart page.

Reply:
noted, will do this in the following pr.